app/helpers/visit_helper.rb

```diff
@@ -46,7 +46,7 @@ def prison_slot_anomalies
   end
 
   def prison_address(glue='<br>')
-    prison_data['address'].join(glue).html_safe
+    prison_data['address'].map(&method(:sanitize)).join(glue).html_safe
```

Comment:
I don't really like to see `html_safe` called on user input (even if it has been sanitised), but there's a handy method that will already do just this: [safe_join](http://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-safe_join).